cura/CuraApplication.py

```diff
@@ -925,3 +926,8 @@ def addAdditionalComponent(self, area_id, component):
         self._additional_components[area_id].append(component)
 
         self.additionalComponentsChanged.emit(area_id)
+
+    @pyqtSlot(str)
```

Comment:
Ah yes forgot to move the import out of the function. I need this function almost every time I debug qml's :-) (on my computer console.log disappears...)